app/javascript/components/navbar.js

```diff
@@ -4,7 +4,7 @@ const initUpdateNavbarOnScroll = () => {
   const logoBefore = document.querySelector("#logo-before")
   const logoAfter = document.querySelector("#logo-after")
 
-  if (navbar) {
+  if (navbar && header && logoBefore && logoAfter) {
```

Comment:
@gitovska  What do you need scroplling here? Is this sth that can be solved with css?